### PR TITLE
[WBH] Plesk

### PR DESCRIPTION
**Pull request by bene2k1**
### Your checklist for this pull request

- [ ] Name your pull request according to the [contribution guidelines](https://github.com/scaleway/docs-content/blob/main/docs/CONTRIBUTING.md).

### Description

Please describe what you added or changed.

### CODE REVIEW

**Comment by bene2k1** [comment]
Quickstart cPanel:
screencapture-localhost-9000-en-docs-managed-services-webhosting-quickstart-2024-04-30-11_58_27

Quickstart Plesk:
screencapture-localhost-9000-en-docs-managed-services-webhosting-quickstart-2024-04-30-11_59_28

File: managed-services/webhosting/how-to/manage-webhosting.mdx [hunk 1]
```diff
@@ -24,15 +24,16 @@ You can access the cPanel interface from the Scaleway console to autonomously ma
...
 2. Click <Icon name="seeMore" /> or the domain name of the Web Hosting you want to configure. The **Hosting information** page displays.
-3. Click **Dashboard** next to **Link to cPanel** to open the cPanel interface in a new tab.
+3. Click **Dashboard** next to **Link to control panel** to open the cPanel interface in a new tab.
     <Lightbox src="scaleway-webhosting-cpanel.webp" alt="" />
     <Message type="tip">
-        Alternatively, you can reach cPanel via the **cPanel** button in the top right corner of the **Hosting information** section.
+        Alternatively, you can reach the control panel via the **Control panel dashboard** button in the top right corner of the **Hosting information** section.
     </Message>
-4. Enter your cPanel credentials and click **Log in** to log into cPanel.
+4. Enter your crontol panel credentials and click **Log in** to log into cPanel.
```
Comment (jcirinosclwy) [inline managed-services/webhosting/how-to/manage-webhosting.mdx:32]:
```suggestion
4. Enter your control panel credentials and click **Log in** to log into cPanel.
```

File: managed-services/webhosting/how-to/order-webhosting.mdx [hunk 2]
```diff
@@ -21,24 +21,22 @@ This page shows how to create a Scaleway Web Hosting plan. Web Hosting is a mana
...
+1. Click **Web Hosting** in the **Managed Services** section of the side menu. This takes you to the Web Hosting dashboard.
+   - If you already have Web Hosting plans, they will be listed. Click the **+ Order Web Hosting** button to add more plans.
+2. Click **Order Web Hosting** to start the creation process.
+3. Choose a **region** and a **control panel** for your Web Hosting, then click **Configure Web Hosting**.
+    <Message type="tip">
+      Unsure which control panel to choose? Check out our documentation on [Understanding the differences between cPanel and Plesk](/managed-services/webhosting/reference-content/understanding-differences-cpanel-plesk/).
+    </Message>
+4. Follow these steps to configure your Web Hosting:
+   - Enter your **domain name**. Remember, it must be pre-registered and you must be the registrant. Need to register one? [Read our documentation how to register a domain name](/network/domains-and-dns/how-to/register-internal-domain/).
```
Comment (jcirinosclwy) [inline managed-services/webhosting/how-to/order-webhosting.mdx:32]:
```suggestion
   - Enter your **domain name**. Remember, it must be pre-registered and you must be the registrant. Need to register one? [Read our documentation on how to register a domain name](/network/domains-and-dns/how-to/register-internal-domain/).
```

File: managed-services/webhosting/how-to/order-webhosting.mdx [hunk 3]
```diff
@@ -21,24 +21,22 @@ This page shows how to create a Scaleway Web Hosting plan. Web Hosting is a mana
...
+   - Choose a **hosting plan** based on your needs.
+   - Decide on activating the **backup option**:
```
Comment (jcirinosclwy) [inline managed-services/webhosting/how-to/order-webhosting.mdx:34]:
```suggestion
   - Decide if you want to enable the **backup** feature:
```

File: managed-services/webhosting/how-to/order-webhosting.mdx [hunk 4]
```diff
@@ -21,24 +21,22 @@ This page shows how to create a Scaleway Web Hosting plan. Web Hosting is a mana
...
+     - Enable for daily automated backups of your website, emails, and databases. Retains backups for three days, with a full weekly backup retained for two months.
+   - Choose whether to order a dedicated IP for your Web Hosting, which can improve email reputation and SSL setup.
+   - Enter your **email address**. We'll send your control panel password there.
```
Comment (jcirinosclwy) [inline managed-services/webhosting/how-to/order-webhosting.mdx:37]:
```suggestion
   - Enter your **email address**. We will send your control panel password there.
```

**Review by jsudraud — APPROVED** [review]

**Pull request merged**